sdk/agrifood/agrifood-farming-rest/samples-dev/createFarmer.ts

```diff
@@ -34,7 +34,8 @@ async function main() {
     throw result.body.error;
   }
 
-  console.log(`Created Farmer: ${result.body.name}`);
+  const farmer = <FarmerOutput>result.body;
```

Comment:
you shouldn't need to cast to `<FarmerOutput>` if you `import isUnexpected from "./isUnexpected"` and change line 33 for

```ts
if (isUnexpected(result)) {
```

Reply:
Removed casting to PartyOutput